sql-cli/sql_cli/project.py

```diff
@@ -82,7 +82,7 @@ def _initialise_global_config(self) -> None:
         Initialises global config file that includes configuration to be shared across environments including the
         airflow config.
         """
-        config = Config(environment=GLOBAL_CONFIG, project_dir=self.directory)
+        config = Config(environment=DEFAULT_ENVIRONMENT, project_dir=self.directory)
```

Comment:
@pankajkoti why is `environment=DEFAULT_ENVIRONMENT` for the "global" configuration?

Reply:
The Config instance has environment as a required key. We can set it to anything as it does not have any effect for the global config keys.

Reply:
> We can set it to anything as it does not have any effect

^this is huge code smell in my opinion.